src/components/CategoryListItem.js

```diff
@@ -1,47 +1,60 @@
 import React from "react";
-import Router from "react-router";
+import Router, { Link } from "react-router";
 import mui from "material-ui";
-import HeaderWithChevron from "./HeaderWithChevron";
+import icons from "../icons";
 
 export default class CategoryListItem extends React.Component {
 
+    href(): string {
+        return this.context.router.makeHref(
+            "category",
+            {categoryName: this.props.category.key},
+            {}
+        );
+    }
+
     render(): React.Element {
+        var Icon = this.props.category.icon || icons.House;
+
         return (
-            <Router.Link
-                to="category"
-                params={{
-                    categoryName: this.props.categoryKey,
-                }}
-                style={{
-                    textDecoration: "none",
-                    fontSize: 24,
-                }}
+            <mui.ListItem
+                href={this.href()}
+                primaryText={this.props.category.name}
+                secondaryText={this.props.category.byline}
+                secondaryTextLines={2}
+                leftAvatar={
+                    <mui.Avatar
+                        backgroundColor={mui.Styles.Colors.transparent}
+                        color="#a21e78"
```

Comment:
I can't think of an obvious way to share the colors between the `mui` inline styles and the scss files. 

I can duplicate the config, or maybe do some rewriting hax with webpack, but none of the options are pleasant.